src/core_plugins/kibana/public/discover/embeddable/search_embeddable.js

```diff
@@ -36,13 +36,19 @@ export class SearchEmbeddable extends Embeddable {
   pushContainerStateParamsToScope() {
     // If there is column or sort data on the panel, that means the original columns or sort settings have
     // been overridden in a dashboard.
+
     this.searchScope.columns = this.customization.columns || this.savedSearch.columns;
     this.searchScope.sort = this.customization.sort || this.savedSearch.sort;
     this.searchScope.sharedItemTitle = this.panelTitle;
+
+    this.searchScope.searchSource.getParent().set('filter', this.filters);
```

Comment:
I think you are losing the time range filter. Unfortunately the passed down time range is not being picked up by saved searches yet.  Was something I was planning to get to at some point... looks like you are going to beat me to it though!

<img width="1301" alt="screen shot 2018-05-21 at 1 26 51 pm" src="https://user-images.githubusercontent.com/16563603/40320774-bd1bf2ca-5cfa-11e8-91e3-3d5055d1b8a7.png">

Reply:
Why setting it on the parent and not `this.searchScope.searchSource`? Could you clarify a bit what search source is which in this case?

Reply:
we created a searchsource 'stack'  (check below) where timeRangeSearchSource is the `root` and filterSearchSouce inherits from it and our saved searchSource inherits from the filterSearchSource

- savedSearchSource will have filters and query saved with the search 
- filterSearchSource will get dashboard filters and query
- timeRangeSearchSource will get the time range filter

we do this as we don't want to handle merging of filters and queries (if we would have only one level) and we let the searchsource handle it.

Reply:
Couldn't we change the inheritance order of the search sources to be: `timeRangeSearchSource` ← `savedSearchSource` ← `filterSearchSource` (i.e. put the saved search source in the middle).

That way we could safely use `this.searchScope.searchSource` instead of needing this kind of implementation detail based call to `getParent()`? Also for the actual request, the inheritance order of those three shouldn't matter?